libmscore/synthesizerstate.cpp

```diff
@@ -54,7 +55,15 @@ void SynthesizerState::read(XmlReader& e)
                   else
                         e.unknown();
                   }
-            push_back(group);
+            tempGroups.push_back(group);
+            }
+
+      if (int(tempGroups.size()) > 0) {
```

Comment:
If there were only `<Synthesizer></Synthesizer` tags, we don't want to override the default, so we have to check whether we've actually read something before clearing the state and setting it to the new, read one.